### ISSUE TITLE
Cannot perform createJob

### ISSUE THREAD

**Issue by Overdrivr** [Paperspace/paperspace-python#35]
Calling paperspace.jobs.create fails, apparently the `time` module is not imported.

```
    return paperspace.jobs.create({
  File "c:\users\remib\.virtualenvs\nexture-studio-vt35otws\lib\site-packages\paperspace\jobs.py", line 159, in create
    job = waitfor({'jobId': jobId, 'state': 'Running'})
  File "c:\users\remib\.virtualenvs\nexture-studio-vt35otws\lib\site-packages\paperspace\jobs.py", line 139, in waitfor
    time.sleep(5)
NameError: name 'time' is not defined
```

**Comment by BartoszCki** [comment]
Thanks for pointing it out @Overdrivr 
This issue is fixed in release 0.0.19a0 - it's an alpha version of experiments support but creating jobs wasn't affected by this change and should be safe to use.

**Comment by BartoszCki** [comment]
To install 0.0.19a0 (or any latest version including alphas and betas) use `pip install paperspace --pre`
